README.md

```diff
@@ -98,32 +98,38 @@ useTimeFormat | 'HHmm' (according to [momentjs date format](https://momentjs.com
-Merlin            | UbpCrsAdapter.CRS_TYPES.merlin   |                            | 
-MyJack / Jack+    | UbpCrsAdapter.CRS_TYPES.bewotec  | token                      | '0123456789abcdef'
-|                 |                                  | dataBridgeUrl              | 'example://url.where-the-adapter/can-get-the-crs-data/when-not-in-http-context'
-Cosmo / CosmoNaut | UbpCrsAdapter.CRS_TYPES.traffics | dataSourceUrl              | 'example://url.where-the-crs/can-get-the-adapter-data'
-|                 |                                  | environment                | \<environment\>
-|                 |                                  | exportId                   | '0123-456789-abcdef'
+CRS               | connectionType                    | connectionOptions          | example
+:---              | :---                              | :---                       | :---
+CETS              | UbpCrsAdapter.CRS_TYPES.cets      |                            | 
+TOMA (old)        | UbpCrsAdapter.CRS_TYPES.toma      | providerKey                | 'ABC'
+TOMA (new)        | UbpCrsAdapter.CRS_TYPES.toma2     | externalCatalogVersion (*) | '20.5'
+|                 |                                   | connectionUrl              | 'https://url-to-amadeus-selling.plattform'
+|                 |                                   | popupId                    | 'popup_id0123456789abcdef'
+Merlin            | UbpCrsAdapter.CRS_TYPES.merlin    |                            | 
+MyJack            | UbpCrsAdapter.CRS_TYPES.myjack    | token                      | '0123456789abcdef'
+|                 |                                   | dataBridgeUrl              | 'example://url.where-the-adapter/can-get-the-crs-data/when-not-in-http-context'
+Jack+             | UbpCrsAdapter.CRS_TYPES.jackplus  | token                      | '0123456789abcdef'
+Cosmo             | UbpCrsAdapter.CRS_TYPES.cosmo     | dataSourceUrl              | 'example://url.where-the-crs/can-get-the-adapter-data'
+|                 |                                   | environment                | \<environment\>
+|                 |                                   | exportId                   | '0123-456789-abcdef'
+CosmoNaut         | UbpCrsAdapter.CRS_TYPES.cosmonaut | dataSourceUrl              | 'example://url.where-the-crs/can-get-the-adapter-data'
+|                 |                                   | environment                | \<environment\>
+|                 |                                   | exportId                   | '0123-456789-abcdef'
 
 (*) optional
 
 For some connections you need credentials or other connection data,
 which you can set in the `connectionOptions`.
 
-**[TOMA 2]** _**connectionUrl** is needed, when the adapter is not directly used in the first child window of the TOMA application_
+**[TOMA 2]** _connectionUrl_ is needed, when the adapter is not directly used in the first child window of the TOMA application_
```

Comment:
Looks like this line is missing a `_` somewhere (or the last one should be a period instead)

Reply:
hm, you are right. it's lost somewhere.